recipes/libcap/all/patches/2.73/0001-libcap-Remove-hardcoded-fPIC.patch

```diff
@@ -0,0 +1,26 @@
+From 8b86ac7b057ce529900969ead958a9ca7ae85b3c Mon Sep 17 00:00:00 2001
+From: Sergey Bobrenok <bobrofon@gmail.com>
+Date: Sun, 22 Dec 2024 20:30:31 +0700
+Subject: [PATCH 1/2] libcap: Remove hardcoded -fPIC
+
+Signed-off-by: Sergey Bobrenok <bobrofon@gmail.com>
+---
+ libcap/Makefile | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/libcap/Makefile b/libcap/Makefile
+index 5f5c601..02be223 100644
+--- a/libcap/Makefile
++++ b/libcap/Makefile
+@@ -19,7 +19,7 @@ PSXFILES=../psx/psx ../psx/wrap/psx_wrap
+ PSXMAGICOBJ=psx_magic.o
+ 
+ # Always build libcap sources this way:
+-CFLAGS += -fPIC -D_LIBPSX_PTHREAD_LINKAGE
++CFLAGS += -D_LIBPSX_PTHREAD_LINKAGE
```

Comment:
Hi @bobrofon, Why is this line added if nothing similar was added in the previous patch?  I couldn’t find anything 
 upstream—could you help me? 😄 

https://github.com/conan-io/conan-center-index/blob/master/recipes/libcap/all/patches/2.57/0001-libcap-Remove-hardcoded-fPIC.patch#L21

Reply:
In the previous patch, the `-fPIC` flag was removed from `CFLAGS`. In this patch, the `-fPIC` flag is also removed from `CFLAGS`. Honestly, I don't see any difference.

```diff
-CFLAGS += -fPIC
```
vs
```diff
-CFLAGS += -fPIC -D_LIBPSX_PTHREAD_LINKAGE
+CFLAGS += -D_LIBPSX_PTHREAD_LINKAGE
```

Reply:
The associated change in the upstream: https://git.kernel.org/pub/scm/libs/libcap/libcap.git/diff/libcap/Makefile?id=libcap-2.73&id2=libcap-2.70
```diff
-CFLAGS += -fPIC
+CFLAGS += -fPIC -D_LIBPSX_PTHREAD_LINKAGE
```